### PR TITLE
[IMP] l10n_es: add DUA exempt tax and its mapping

### PR DESCRIPTION
**Pull request by Tisho99**
Description of the issue/feature this PR addresses: Add DUA exempt tax and its mapping

Current behavior before PR: Only one way to manage DUA

Desired behavior after PR is merged: Added an alternative to manage DUA with an innocuous tax and its mapping

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Tisho99** [comment]
@HaraldPanten

**Comment by HaraldPanten** [comment]
@jco-odoo After our conversation in the Spanish OCA Days, we agreed on creating a "dummy" tax in order to manage the import of goods (with DUA) in a better way.

Could you have a look?

@pedrobaeza your review will be welcome, as well.

**Comment by pedrobaeza** [comment]
The CLA addition should be handled in another PR IMO, and maybe for lower versions. The rest seems OK in a glance. To be checked on runbot.

**Comment by jco-odoo** [comment]
I don't mind merging here and having a separate PR in 15 for the CLA (it could be a separate commit with [CLA]).  I have also started the l10n build to see if it that is green.

**Comment by mart-e** [comment]
> The CLA addition should be handled in another PR IMO, and maybe for lower versions

Honnestly I prefer if it’s in the same PR (avoid noise of PR just for CLA or contributors signing CLA but never actually contributing) but if you want to sign in a lower version than the one you are making a PR, it’s fine.

**Comment by Tisho99** [comment]
Hi @mart-e 

This is the CLA commit. #165942

I prefer that one to be merged because it affects more versions. 

Do i need to change or close any of the PRs?

Thank you!

**Comment by HaraldPanten** [comment]
@mart-e @jco-odoo I think this PR can be moved forward 🤙🏼

**Comment by Tisho99** [comment]
Branch rebased with my CLA validation

File: addons/l10n_es/data/template/account.fiscal.position-es_common.csv [hunk 1]
```diff
@@ -33,15 +33,15 @@
 "","","","","","","","","","","","account_common_7050","account_common_7051"
 "fp_extra","4","Extra-community","Régimen Extracomunitario","Règim extracomunitari","1","","","","account_tax_template_p_iva0_s_bc","account_tax_template_p_iva0_ibc","",""
 "","","","","","","","","","account_tax_template_p_iva0_s_sc","account_tax_template_p_iva0_isc","",""
-"","","","","","","","","","account_tax_template_p_iva4_bc","","",""
+"","","","","","","","","","account_tax_template_p_iva4_bc","account_tax_template_p_iva0_ibc_group","",""
```
Comment (jco-odoo) [inline addons/l10n_es/data/template/account.fiscal.position-es_common.csv:36]: This is wrong.  This should map to the exento tax...

Comment (Tisho99) [reply addons/l10n_es/data/template/account.fiscal.position-es_common.csv:36]: That is the Exempt DUA tax, maybe the external_id name is wrong or i am misunderstanding something?

Comment (jco-odoo) [reply addons/l10n_es/data/template/account.fiscal.position-es_common.csv:36]: Oh wow, you are right.  It is just that the account_tax_template_p_iva0_ibc_group name is confusing.  As it is not even a tax group.

Comment (Tisho99) [reply addons/l10n_es/data/template/account.fiscal.position-es_common.csv]: I have changed it's name to account_tax_template_p_iva0_dua_exempt, does it seem better?

Comment (jco-odoo) [reply addons/l10n_es/data/template/account.fiscal.position-es_common.csv]: Yes.  It is not really iva0 as well, so I wonder if we should remove the _iva0 part.

File: addons/l10n_es/data/template/account.tax-es_common.csv [hunk 2]
```diff
@@ -585,6 +585,10 @@
 "","","","","","","","","","","","","","","","","","tax","","invoice","","","",""
 "","","","","","","","","","","","","","","","","","base","","refund","","","",""
 "","","","","","","","","","","","","","","","","","tax","","refund","","","",""
+"account_tax_template_p_iva0_dua_exempt","","Exempt DUA","","purchase","Exempt DUA","DUA exento","DUA exempt","0.0","percent","tax_group_dua_exempt","","dua","","","consu","","base","","invoice","","100","",""
```
Comment (jco-odoo) [inline addons/l10n_es/data/template/account.tax-es_common.csv:588]: This should not be dua type, but "ignore"

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**